wet-boew/charts/index.json-ld

```diff
@@ -0,0 +1,46 @@
+{
+	"@context": {
+		"@version": 1.1,
+		"dct": "http://purl.org/dc/terms/",
+		"title": { "@id": "dct:title", "@container": "@language" },
+		"description": { "@id": "dct:description", "@container": "@language" },
+		"modified": "dct:modified"
+	},
+		"title": {
+		"en": "Charts and Graphs",
+		"fr": "Graphiques"
+	},
+	"description": {
+		"en": "Dynamically generates charts and graphs from table data",
+		"fr": "Génère dynamiquement des graphiques et des diagrammes à partir des données de tableaux"
+	},
+	"modified": "2024-05-16",
+	"componentName": "charts",
+	"status": "stable",
+	"pages": {
+		"examples": [
```

Comment:
Please add the other examples
* Customization
* Customizing pie charts
* Labels and reference values
* Specific test cases